kernel/dev/graphics/font.hpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file font.hpp
+ * @author Keeton Feavel (keetonfeavel@cedarville.edu)
+ * @brief 
+ * @version 0.1
+ * @date 2021-07-25
+ * 
+ * @copyright Copyright the Panix Contributors (c) 2021
+ * 
+ */
+#pragma once
+#include <dev/graphics/framebuffer.hpp>
+#include <stdint.h>
+
+#define FONT_WIDTH 8
+#define FONT_HEIGHT 8
+
+namespace Graphics {
+namespace Font {
+
+/**
+ * @brief Draw the character at the given coordinates in the given color.
+ *
+ * @param c Character to draw
+ * @param x Horizontal position
+ * @param y Vertical position
+ * @param fore Character hex color
+ */
+void Draw(char c, uint32_t x, uint32_t y, uint32_t fore);
```

Comment:
Should the font be locked, or should whatever is accessing the font (console, etc) be required to lock?

Reply:
Yeah really anything that writes to the same buffer should be locked.

Reply:
This will be resolved when #275 is finished since the font will be written to a specified `Screen` object. That way the font data isn't written to the master framebuffer.